msteams-platform/tabs/what-are-tabs.md

```diff
@@ -129,12 +129,12 @@ A custom tab is declared in the app manifest of your app package. For each webpa
 
 Whether you choose to expose your tab within the channel or group, or personal scope, you must present an <iframe\> HTML [content page](~/tabs/how-to/create-tab-pages/content-page.md) in your tab. For static tabs, the content URL is set directly in your Teams [app manifest](../resources/schema/manifest-schema.md#statictabs) by the `contentUrl` property in the `staticTabs` array. Your tab's content is the same for all users.
 
-> [!Note]
-> Teams app doesn't recognize sub iframes. Therefore, it'll not load if there is an iframe within the tab app.
+> [!NOTE]
+> Teams app can't use native plugins when it runs inside sandboxed iframes.
```

Comment:
NIT: I think this should say:
"Teams apps can't use native plugins because they run inside sandboxed iframes."

Reply:
Updated as suggested.